biomage/stage/stage.py

```diff
@@ -258,7 +258,7 @@ def get_sandbox_id(templates, manifests, all_experiments, auto):
                 )
             )
         elif SANDBOX_NAME_REGEX.match(sandbox_id) and len(sandbox_id) <= 26:
-            return sandbox_id
+            return sandbox_id.lower()
```

Comment:
I'm not familiar with this code, doing this will cause the command to fail if it has uppercase letters?

Reply:
No, the problem is that the match method returns a match object or nothing if it does not match. A string like "alex-upgrade-R-pipeline242" would return a match object from the a to the - before the R. That is enough to get into the if block, and the function return the complete id

Then, the GitHub action fails, because the name contains an uppercase letter.

The .lower here is not necessary. I forgot to remove it.

Reply:
```suggestion
            return sandbox_id
```